faq/programming.po

```diff
@@ -1541,14 +1907,14 @@ msgid ""
 "it's usually a lot slower than using Python lists."
 msgstr ""
 "Si vous voulez pouvoir modifier les éléments, utilisez une liste plutôt "
-"qu'un tuple. Ici la version équivalente au *car* de Lisp est "
-"``lisp_list[0]`` et l'équivalent à *cdr* est ``list_lip[1]``. Ne faites ceci "
-"que si vous êtes réellement sûr d'en avoir besoin, cette méthode est en "
-"générale bien plus lente que les listes Python."
+"qu'un tuple. Ici la version équivalente du *car* de Lisp est "
+"``lisp_list[0]`` et l'équivalent de *cdr* est ``list_lip[1]``. Ne faites "
```

Comment:
```suggestion
"``lisp_list[0]`` et l'équivalent de *cdr* est ``lisp_list[1]``. Ne faites "
```
Par ailleurs, l'équivalent de *cdr* n'est-il pas lisp_list[1:] ? -> à remonter upstream

Reply:
Non non cdr est bien lisp_list[1], car¹ leur liste est :

    lisp_list = ("like",  ("this",  ("example", None) ) )

donc lisp_list[1] donne bien `("this",  ("example", None) )` (car c'est "le premier" et cdr "le reste" en lisp)

1: oui j'ai fais exprès.